### DISCUSSION TITLE
🍎微星 B460M HD630 10700 RX580/RX590 HACKINTOSH黑苹果

### DISCUSSION THREAD

**Discussion by giscus-kuokuo-io** [nagaame/giscus-discussions#12] [Announcements]
# 🍎微星 B460M HD630 10700 RX580/RX590 HACKINTOSH黑苹果

硬件列表
项目名称价格CPUIntel 10th CORE 107002559主板微星迫击炮 MAG B460M MORTAR WiFi内存芝奇 幻光戟 16GB * 2
DDR4 3200800显卡核显 HD6300硬盘三星 970EVO Plus SSD 1TB1019硬盘2Intel 760P SSD0电源安钛克
NE650W469散热器追风者 12DX 黑白版249机箱风扇追风者 12cm RGB * 3129机箱追风者 MATX 410449无线网卡
BCM94360CS2 PCIE200独立显卡蓝宝石 RX5901500跑分
CPU

GPU
OpenCL(rx590)

Metal(rx590)

截图
见仓库文件夹
[https://github.com/nagaame/MSI-B460M-MORTAR-WIFI-10700-HACKINTOSH/tree/master/screenshot]
二进制文件见release
[https:

https://kuokuo.io/2022/04/06/msi-b460m-mortar-wifi-10700-hackintosh/

**Comment by nagaame** [comment]
testing
